VsViewer/vs_calc/cpt_vs_correlations.py

```diff
@@ -45,18 +45,55 @@ def mcgann_2018(cpt: CPT):
     return VsMcGann2, Vs_SD
 
 
-def andrus_2007(cpt: CPT):
+def andrus_2007_holocene(cpt: CPT):
     """
-    CPT-Vs correlation developed by Andrus et al. (2007).
-    qt in kPa
+    CPT-Vs correlation developed by Andrus et al. (2007) for Holocene-age soils (Equation 5).
+    qt in kPa.
```

Comment:
Could you run copilot over this file and get it to produce numpy-style docstrings for the functions in this module?